ci-operator/step-registry/hypershift/aws/e2e/hypershift-aws-e2e-workflow.yaml

```diff
@@ -0,0 +1,36 @@
+workflow:
+  as: hypershift-aws-e2e
+  documentation: |-
+    The HyperShift aws e2e workflow executes tests against a new ephemeral
+    HyperShift cluster. Administrative access to the control plane is provided
+    via the `KUBECONFIG` environment variable.
+
+    Note that tests will execute once the control plane is ready, regardless of
+    whether nodes have been requested, have joined, or are available.
+
+    The control plane is hosted on a dedicated standalone OCP cluster running
+    HyperShift in the `osd-hypershift` AWS account and is destroyed after tests
+    complete.
+
+    The HyperShift launch capability is currently supported by the HyperShift
+    team. For now, please direct all questions and comments to:
+
+    - Alberto Lamela (agarcial@redhat.com)
+    - Seth Jennings (sjenning@redhat.com)
+    - Alvaro Aleman (alvaroaleman@redhat.com)
+    - Cesar Wong (cewong@redhat.com)
+
+    Learn more about HyperShift here: https://github.com/openshift/hypershift
+
+    Track HyperShift's development here: https://issues.redhat.com/projects/HOSTEDCP
+  steps:
+    post:
+    - chain: hypershift-dump
+    - chain: hypershift-aws-destroy
+    - chain: hypershift-aws-destroy-nested-management-cluster
+    test:
+    - chain: hypershift-aws-run-e2e
+    pre:
+    - ref: ipi-install-rbac
+    - chain: hypershift-aws-setup-nested-management-cluster
+    - chain: hypershift-install
```

Comment:
The hypershift-install chain should be modified to use `make ci-install-hypershift-private` instead of explicit commands:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/hypershift/install/hypershift-install-chain.yaml

Reply:
agreed, though this step is used by the aws conformance workflow as well, I was trying to keep blast radius of the PR as small as possible.

Reply:
also need to make sure install command uses the HYPERSHIFT_RELEASE_LATEST build from the source code